### ISSUE TITLE
No translation file found for domain: 'squeeze-alexa' 

### ISSUE THREAD

**Issue by bgeneto** [declension/squeeze-alexa#46]
I'm getting the following error from 'gettext.py' while running bin/local_test.py:
```
IOError: [Errno 2] No translation file found for domain: 'squeeze-alexa' 
```
I'm using LOCALE = 'en_US' in 'settings.py' because I've configured my Alexa Skill to English (us) in AWS. But also tried 'en_GB' without success.
My 'locale' directory contains only one file: 
```
ls locale/en_GB/LC_MESSAGES/
squeeze-alexa.po
```
Any help is welcome.

**Comment by declension** [comment]
Hi. I guess you're using latest from Git `master`. That's recently [been internationalised](#44), and I guess I haven't finished all the documentation / scripting that results in this. If you'd prefer, try the [1.1 release version](https://github.com/declension/squeeze-alexa/releases/tag/v1.1) which predates all this.

That message means `gettext` can't find compiled translations (`.mo` files) for your locale and 'domain', here `squeeze-alexa`). 

To fix it you can run the script included:

```bash
bin/compile-translations
```
for which you'll need gettext tools installed on your OS if you haven't got it (e.g. `sudo apt install gettext` on Ubuntu)

I'll try to document this better

**Comment by declension** [comment]
If you want to "translate" squeeze-alexa to `en_US` then that's a bit harder, but I've added scripting to help with this too. And there _is_ some documentation for that... under [the translation section](https://github.com/declension/squeeze-alexa/blob/master/docs/CONTRIBUTING.md#i-have-a-new-language)

**Comment by declension** [comment]
(Added a fallback to allow `en_US` by default as a fix for #49 )
